datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -183,6 +183,27 @@ def resolve_db_host(db_host):
     return db_host
 
 
+def get_agent_host_tags():
+    """
+    Get the tags from the agent host and return them as a list of strings.
+    """
+    result = []
+    host_tags = datadog_agent.get_host_tags()
+    if not host_tags:
+        return result
+    try:
+        tags_dict = json.loads(host_tags) or {}
+    except Exception as e:
+        logger.warning("Failed to parse tags: %s", host_tags)
+        return result
```

Comment:
Sorry, one more comment/question here. Is failing silently (with only a log) the expected behavior we want here? I'm trying to think about how this will be used so it's hard to say for certain, but I can see this causing some nasty bugs. 

Since this is a base package, the contract should be very clear. Will we _ever_ want to handle this error directly in the integrations?

Reply:
Yeah this is a great point. I was going to chat with Zhengda about this tomorrow - I'm not even sure we would hit this case and the experience right now doesn't make sense (would the user have to tell us they see this log and then we'd have to debug it? is it something we could alert on?) I'll talk to Zhengda and Casey tomorrow